src/controllers/commentController.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable class-methods-use-this */
+import Response from '../utils/response';
+import CommentService from '../repositories/commentRepository';
+import UserService from '../repositories/userRepository';
+import RequestService from '../repositories/requestRepository';
+
+class CommentController {
+  /**
+   * Creates a new comment.
+   * @param {object} req request
+   * @param {object} res response
+   * @param {object} next next
+   * @returns {object} response object
+   */
+  static async addComment(req, res, next) {
+    try {
+      const { comment } = req.body;
+      const id = Number(req.params.id);
+      const requestExists = await RequestService.findRequestById(id);
+      if (!requestExists) {
+        const response = new Response(res, 404, 'Request is not found to add a comment');
+        return response.sendErrorMessage();
+      }
+      const user = await UserService.findByUserId(req.userData.id);
+      const addedComment = await CommentService.addComment({
+        user_id: user.id,
+        request_id: id,
+        comment
+      }, {
+        fields: ['user_id', 'request_id', 'comment']
+      });
+      const response = new Response(res, 201, 'comment sucessfully created', addedComment);
+      response.sendSuccessResponse();
+    } catch (error) {
+      return next(error);
+    }
+  }
+
+  /**
+   * gets all comments by request
+   * @param {object} req request
+   * @param {object} res response
+   * @param {object} next next
+   * @returns {object} response
+   */
+  static async getCommentsByRequest(req, res, next) {
+    try {
+      const id = Number(req.params.id);
+      const requestExists = await RequestService.findRequestById(id);
+      if (!requestExists) {
+        const response = new Response(res, 404, 'Request is not found with this ID');
```

Comment:
@KarangwaEmmy **Request is not found with this ID**, which id ? it can be better if you just say ' Request not Found' Thank you